CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changes
 
+## edge-21.5.2
+
+This edge release updates the proxy-init container to check whether the iptables
+rules have already been added, which prevents errors if the proxy-init container
+is restarted. Also, the linkerd cli now has tab completion for kubernetes
+resources, saving you precious keystrokes! Finally, the proxy has been updated
+with several fixes and improvements.
+
+* Added instructions to `build.md` for using a locally built proxy
+  (thanks @jroper!)
+* Added support for kubernetes resource aware tab completion
+* Updated `proxy-init` to skip configuring firewall if rules exists
+* Fixed viz uninstall to delete all RBAC objects (thanks @aryan9600!)
+* Improved diagnostics for rejected profile discovery
+* Added proxy logic to set the l5d-client-id header
+* Reduced proxy resource usage when there are no profiles
+* Updated proxy to fail connections when HTTP detection fails
+* Changed proxy to assume meshed connections are HTTP/2
+* Updated the proxy to require the `l5d-dst-override` header
```

Comment:
This is very specific to outbound requests for proxy that is in ingress-mode. The header is not required otherwise.

Reply:
Thanks for clarifying, I updated the note to specify outbound requests in ingress mode